UnitTesting-master/src/test/java/seleniumMail/Tests/YandexMailTest.java

```diff
@@ -0,0 +1,58 @@
+package seleniumMail.Tests;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import seleniumMail.Helpers.PropertiesUtil;
+import seleniumMail.Pages.InboxPage;
+import seleniumMail.Pages.LoginPage;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class YandexMailTest {
+
+    private static final String USERNAME = "user1";
+    private static final String PASSWORD = "password1";
+    private static final String YANDEX_MAIL_URL = "mail.url";
+
+    private static String user = PropertiesUtil.get(USERNAME);
+    private static String password = PropertiesUtil.get(PASSWORD);
+    private static String url = PropertiesUtil.get(YANDEX_MAIL_URL);
+
+    private WebDriver driver;
+
+    @BeforeEach
+    void setup() {
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+    }
+
+    @Test
+    public void logInTest() {
+        driver.navigate().to(url);
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.logIn(user, password);
+        InboxPage inboxPage = new InboxPage(driver);
```

Comment:
When you will update page objects and create singleton for WebDriver, your code will look like:
```
LoginPage loginPage = new LoginPage();
InboxPage inboxPage = loginPage.logIn(user, password);
```